xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -1034,6 +1035,11 @@ void CXBMCApp::onReceive(CJNIIntent intent)
     if (m_playback_state & PLAYBACK_STATE_VIDEO)
       CApplicationMessenger::GetInstance().PostMsg(TMSG_GUI_ACTION, WINDOW_INVALID, -1, static_cast<void*>(new CAction(ACTION_STOP)));
   }
+  else if (action == "android.media.AUDIO_BECOMING_NOISY")
+  {
+    if (m_playback_state & PLAYBACK_STATE_AUDIO)
+      CApplicationMessenger::GetInstance().PostMsg(TMSG_GUI_ACTION, WINDOW_INVALID, -1, static_cast<void*>(new CAction(ACTION_PAUSE)));
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting

Reply:
Updated by using clang-format to format this chunk of code.